.github/assign-by-files.yml

```
@@ -83,7 +83,7 @@
 #   - TomShawn
 #   - 
 
-# 'system-tables/**/*.md':
+# 'information-schema/**/*.md':
 #   - TomShawn
 #   - 
```

```suggestion
'information-schema/**/*.md':
  - TomShawn
  - nullnotnil
```
Do you wanna review `information-schema` related PRs in the future? @nullnotnil